checks/raw/fuzzing.go

```diff
@@ -48,14 +78,28 @@ func Fuzzing(c *checker.CheckRequest) (checker.FuzzingData, error) {
 	if usingOSSFuzz {
 		fuzzers = append(fuzzers,
 			checker.Tool{
-				Name: "OSS-Fuzz",
+				Name: FuzzNameOSSFuzz,
 				URL:  asPointer("https://github.com/google/oss-fuzz"),
 				Desc: asPointer("Continuous Fuzzing for Open Source Software"),
 				// TODO: File.
 			},
 		)
 	}
 
+	usingFuzzFunc, files, e := checkFuzzFunc(c)
+	if e != nil {
+		return checker.FuzzingData{}, fmt.Errorf("%w", e)
+	}
+	if usingFuzzFunc {
+		fuzzers = append(fuzzers,
+			checker.Tool{
+				Name: FuzzNameUserDefinedFunc,
+				URL:  asPointer("https://en.wikipedia.org/wiki/Fuzzing"),
```

Comment:
For Go fuzzing, point to the official page/documentation of Go fuzzing.

Reply:
my concern is the same as the above one